packages/taquito/src/operations/operation-errors.ts

```diff
@@ -74,6 +75,7 @@ export type MergedOperationResult = OperationResultTransaction &
   OperationResultTxRollupOrigination &
   OperationResultTxRollupSubmitBatch &
   OperationResultTransferTicket &
+  Partial<OperationResultSmartRollupOriginate> &
```

Comment:
why partial?

Reply:
because otherwise it is expected that other operations have items that are specific to smartRollupOriginate

Reply:
otherwise the other types are expected to have the same expected key item pairs